.github/workflows/clients-node-build.yml

```diff
@@ -36,6 +36,15 @@ jobs:
       - uses: actions/checkout@v3
       - run: ./scripts/test_install_on_${{ matrix.distro }}.sh
 
+  test_macos_install:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: npm install -g npm@8
+      - run: npm install
+      - run: npm pack
+      - run: dir=$(pwd) && mkdir /tmp/test && cd /tmp/test && npm install $dir/tigerbeetle-node-*.tgz && node -e 'require("tigerbeetle-node"); console.log("SUCCESS!")'
```

Comment:
@matklad since you've got another branch open for debugging, I had to make some more changes to this to get the macos build working

Reply:
It needs to install and pack and then the real test is installing from the packaged thing.